build.sbt

```diff
@@ -329,6 +329,8 @@ lazy val scaldingCore = module("core").settings(
     "com.twitter" %% "bijection-macros" % bijectionVersion,
     "com.twitter" %% "chill" % chillVersion,
     "com.twitter" %% "chill-algebird" % chillVersion,
+    "com.twitter.elephantbird" % "elephant-bird-cascading2" % elephantbirdVersion,
```

Comment:
@johnynek got it, let's keep elephantbird isolated in commons for now and use combined for VKVS and have separated CombinedSchemas for sequence files as helpers.